src/main/java/me/rayzr522/jsonmessage/JSONMessage.java

```diff
@@ -15,29 +15,21 @@
 import java.lang.invoke.MethodHandles;
 import java.lang.reflect.Constructor;
 import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
 import java.lang.reflect.Method;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Objects;
-import java.util.Vector;
+import java.util.*;
 
-/**
- * This is a complete JSON message builder class. To create a new JSONMessage do
- * {@link #create(String)}
- *
- * @author Rayzr
```

Comment:
why the removal of javadocs? would prefer this remain present

Reply:
Didn't notice that this was removed for some reason. Will re-add it.